Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+    command_results = generate_simple_command_results(object_type=OutputTitle.PROTECTED_HOSTNAME_GROUP.value,
+                                                      object_name=name,
+                                                      action=OutputTitle.CREATED.value,
+                                                      response=response)
+    return command_results
+
+
+def protected_hostname_group_update_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Update a protected hostname.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    validate_protected_hostname_group(args=args)
+    name = args['name']
+    response = client.protected_hostname_update_request(name=name, default_action=args.get('default_action'))
+    command_results = generate_simple_command_results(object_type=OutputTitle.PROTECTED_HOSTNAME_GROUP.value,
+                                                      object_name=name,
+                                                      action=OutputTitle.UPDATED.value,
+                                                      response=response)
+    return command_results
+
+
+def protected_hostname_group_delete_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Delete a protected hostname.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    name = args['name']
+    response = client.protected_hostname_delete_request(name)
+    command_results = generate_simple_command_results(object_type=OutputTitle.PROTECTED_HOSTNAME_GROUP.value,
+                                                      object_name=name,
+                                                      action=OutputTitle.DELETED.value,
+                                                      response=response)
+    return command_results
+
+
+def protected_hostname_group_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Get protected hostname list / object.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    protected_hostname = args.get('name')
+    response = client.protected_hostname_list_request(name=protected_hostname)
+    parsed_data, pagination_message, formatted_response = list_response_handler(
+        client=client,
+        response=response,
+        data_parser=client.parser.parse_protected_hostname_group,
+        args=args,
+        sub_object_id=protected_hostname,
+        sub_object_key='_id' if client == ClientV1.API_VER else 'name')
```

Comment:
client == version? what is checked here? 
Anyway, please use a property or attribute for all these `if` cases.